### ISSUE TITLE
Cobinhood fetch_my_trades wrong side

### ISSUE THREAD

**Issue by blaaaaaaa** [ccxt/ccxt#4300]
Hello,

fetch_my_trades on cobinhood is returning "sell" for a "buy" order and the fetch_my_trades method. Don't have a sell order yet to see if the other side is wrong as well.

- OS:
- Programming Language version:python
- CCXT version:1.18.11
- Exchange:cobinhood
- Method:fetch_my_trades

**Comment by kroitor** [comment]
Hi! Can you please provide your verbose request/response (without your keys), as described here: https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#how-to-submit-an-issue ? 

Related: 

- https://github.com/ccxt/ccxt/pull/3553
- https://github.com/ccxt/ccxt/pull/3514

**Comment by blaaaaaaa** [comment]
I most likely was the maker (on the bid side) and from the data which shows, I think we just cannot know if it was a buy or sell just from the data we are receiving.

Request: GET https://api.cobinhood.com/v1/trading/trades?trading_pair_id=XRP-BTC {'nonce': '1544338807259', 'Authorization': 'xxx', 'User-Agent': 'python-requests/2.18.4', 'Accept-Encoding': 'gzip, deflate'} None

Response: GET https://api.cobinhood.com/v1/trading/trades?trading_pair_id=XRP-BTC 200 {'Date': 'Sun, 09 Dec 2018 07:00:07 GMT', 'Content-Type': 'application/json; charset=utf-8', 'Transfer-Encoding': 'chunked', 'Connection': 'keep-alive', 'Set-Cookie': 'xxx; expires=Mon, 09-Dec-19 07:00:07 GMT; path=/; domain=.cobinhood.com; HttpOnly; Secure', 'Access-Control-Allow-Origin': 'https://cobinhood.com', 'Via': '1.1 google', 'Alt-Svc': 'clear', 'Strict-Transport-Security': 'max-age=15552000; includeSubDomains; preload', 'X-Content-Type-Options': 'nosniff', 'Expect-CT': 'max-age=604800, report-uri="https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct"', 'Server': 'cloudflare', 'CF-RAY': '486597ca4e5cc288-FRA', 'Content-Encoding': 'gzip'} {"success":true,"result":{"limit":50,"page":1,"total_page":1,"total_count":1,"trades":[{"id":"49e82216-c09b-48ec-b811-56bc22fc03e6","maker_side":"bid","trading_pair_id":"XRP-BTC","timestamp":1544273805164,"price":"0.00008245","size":"95.67684192"}]}}

#print(result):

[{'info': {'id': '49e82216-c09b-48ec-b811-56bc22fc03e6', 'maker_side': 'bid', 'trading_pair_id': 'XRP-BTC', 'timestamp': 1544273805164, 'price': '0.00008245', 'size': '95.67684192'}, 'timestamp': 1544273805164, 'datetime': '2018-12-08T12:56:45.164Z', 'symbol': 'XRP/BTC', 'id': '49e82216-c09b-48ec-b811-56bc22fc03e6', 'order': None, 'type': None, 'side': 'sell', 'price': 8.245e-05, 'amount': 95.67684192, 'cost': 0.007888555616304, 'fee': None}]

**Comment by kroitor** [comment]
> I think we just cannot know if it was a buy or sell just from the data we are receiving

Right... What would you suggest to resolve it? Should we just set the `side` field to `undefined` (js) / `None` (py) / `null` (php) for trades returned from `fetchMyTrades` ?

**Comment by blaaaaaaa** [comment]
maybe we are able to get the correct side over the get-order endpoint, but probably not very convinient to call this endpoint for any trade that came back with the first response.

https://cobinhood.github.io/api-public/#get-order

Or maybe get all info in one go from here:
https://cobinhood.github.io/api-public/#get-order-history

**Comment by kroitor** [comment]
> maybe we are able to get the correct side over the get-order endpoint, but probably not very convinient to call this endpoint for any trade that came back with the first response

We'd like to avoid adding that request overhead inside the lib, because not all use cases require that type of operation, it mostly depends on the business logic in userland and it is beyond CCXT for that reason, so we prefer to keep it outside. In other words, if the user needs it – we provide all the necessary means to get orders and trades separately on just the right moment when the user needs it, and then the user can match them together to make the conclusion on `buy/sell` `side` and `makerOrTaker` side.

> Or maybe get all info in one go from here:

That's a different method then )) Check this out: https://github.com/ccxt/ccxt/issues/4301

If any piece of info is not known for sure, we usually avoid trying to guess it. So, I think we will just set a no-value for the `side` field. Are you ok with that?

**Comment by blaaaaaaa** [comment]
Of course thats ok, you are the boss :) I am just new to this. so do I understand correct that fetch_closed_orders will be implemented next version and the user can work out for himself to get the correct infos he needed from that?

**Comment by kroitor** [comment]
> so do I understand correct that fetch_closed_orders will be implemented next version and the user can work out for himself to get the correct infos he needed from that?

Yep, everything is correct, the user can choose which method to call, `fetchOrder`, `fetchOpenOrders`, or `fetchClosedOrders` depending on his needs and work it out from there. It is already implemented in the most recent version 1.18.13.
